packages/teams-js/CHANGELOG.md

```diff
@@ -267,6 +267,7 @@ Tue, 01 Mar 2022 19:50:49 GMT
     - `applyRuntimeConfig` is added
 
 - Promises introduced
+
```

Comment:
FWIW I didn't intentionally put this newline here and I cannot remove it (in VS Code at least -- when I save the file, it auto-inserts the newline). Don't think it hurts leaving it in.